content/tutorial/02-sveltekit/02-routing/06-loading-layout-server-data/README.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Loading Layout Data on the Server
```

Comment:
I don't think we need a separate section here, we can have

* loading page data
* loading layout data
* loading data (page _or_ layout) on the server

Reply:
Something like "we can also move these calls to the server an then safe us the indirection of going through our own `+server.js`, let's rename `+page.js` to `+page.server.js` and `+layout.js` to `+layout.server.js` and adjust the load functions to call the fake database directly?

Reply:
We shouldn't be talking about `+server.js` here at all, it's unnecessarily confusing. Re order, see https://github.com/sveltejs/learn.svelte.dev/pull/105/files#r1038471015

Reply:
That sounds good. I was thinking about the blog example initially, too, but felt it may be too complex and hide the actual teachings. But it sounds like you have a good idea for a simple version. Feel free to shuffle the routing chapter around (although @benmccann wanted to review, too, now, maybe he has a different opinion), getting late here.